src/user_crate/state_generated.rs

```diff
@@ -419,7 +439,7 @@ mod tests {
 
             let generated_lib_rs = generated.lib_rs()?;
             let fixture_lib_rs = parse_quote! {
-                use ::pgx::*;
+                use pgx::*;
```

Comment:
This should remain the same.

Reply:
ooc what's the difference? rustfmt seems to correct it so I wasn't too sure

Reply:
Running `cargo fmt` has never corrected this for me before.
`use ::name::{import, imports};` in practice is rarely needed. However, it is hypothetically possible for a name to be shadowed. `use ::` starts from the "top level" always, so it means that you import from pgx, the crate, and not from, say, a user module defined with the name pgx.

https://doc.rust-lang.org/reference/items/use-declarations.html#use-paths

Reply:
Interesting, cargo fmt and rustfmt seems to work differently?
Tried cargo fmt and got what you mentioned but with rustfmt

```
dev-dsk-hsuchen-2a-9713b625 % git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean

(22-09-09 17:23:20) <0> [~/proj/plrust/src]  
dev-dsk-hsuchen-2a-9713b625 % find . -name "*.rs" -exec rustfmt {} \;

(22-09-09 17:23:24) <0> [~/proj/plrust/src]  
dev-dsk-hsuchen-2a-9713b625 % git diff
diff --git a/src/tests.rs b/src/tests.rs
index d4c213f..4fa4610 100644
--- a/src/tests.rs
+++ b/src/tests.rs
@@ -346,7 +346,7 @@ mod tests {
     #[pg_test]
     #[search_path(@extschema@)]
     fn plrust_one_sleepy_boi() {
-        use std::{time::Duration, thread::sleep};
+        use std::{thread::sleep, time::Duration};
         let moment = Duration::from_secs(2);
         sleep(moment);
 
diff --git a/src/user_crate/state_generated.rs b/src/user_crate/state_generated.rs
index 6e1b7bc..a438085 100644
--- a/src/user_crate/state_generated.rs
+++ b/src/user_crate/state_generated.rs
@@ -127,7 +127,7 @@ impl StateGenerated {
     #[tracing::instrument(level = "debug", skip_all, fields(fn_oid = %self.fn_oid))]
     pub(crate) fn lib_rs(&self) -> eyre::Result<syn::File> {
         let mut skeleton: syn::File = syn::parse_quote!(
-            use ::pgx::*;
+            use pgx::*;
         );
 
         let crate_name = self.crate_name();
```

Reply:
They do not work differently. Rather, `cargo fmt` invokes `rustfmt` with certain flags, which it picks out from reading the Cargo.toml file. The same is also the case with all cargo commands: cargo is rustc's most adept "power user".

Reply:
Ahh thanks for the clarification. Let me make the change